lib/fluent/plugin/filter_process_ucs_syslog.rb

```diff
@@ -187,7 +189,7 @@ def determineMachineId(record, regex)
       begin
         serviceProfile = getServiceProfile(record[ucsHostNameKey], chassisNumber, bladeNumber, 1)
       rescue SecurityError => se
-        record["error"] = se.message
+        record["error"] += "Error getting service profile: #{se.message}"
```

Comment:
Why `+=`? What information is already stored in `record["error"]` at this stage of execution?

Reply:
record["error"] is set to "" near the beginning.

I thought it made more sense since we are appending the etcd error as well, so it is more consistent. Also just in case in the future we might add more errors before this.

Reply:
ack